src/renderer/main.js

```diff
@@ -302,6 +295,7 @@ function backToList () {
     var contentTag = document.querySelector('.content')
     if (contentTag) contentTag.scrollTop = 0
 
+    // TODO dcposch: is this still required with React?
```

Comment:
It's probably still required because we're still using a `requestAnimationFrame` loop. But maybe there's a cleaner way to tell react to redraw with `app.setState(state)`.